### PR TITLE
More utterances

### PR DESCRIPTION
**Pull request by chriswilk**

### CODE REVIEW

**Comment by chriswilk** [comment]
@tomrf1 can you check these?

File: speechAssets/IntentSchema.json [hunk 1]
```diff
@@ -28,12 +25,6 @@
         "intent": "GetIntroNewsIntent"
       },
       {
-        "intent": "GetIntroReviewsIntent"
-      },
-      {
-        "intent": "GetIntroSportIntent"
```
Comment (tomrf1) [inline speechAssets/IntentSchema.json]: `GetIntroSportIntent` is still used

Comment (tomrf1) [inline speechAssets/IntentSchema.json]: I'm wrong about this - we only trigger this intent internally, so it can be removed from here.

**Comment by tomrf1** [comment]
👍

**Pull request merged**